modules/product/css/product.rendered-attributes.css

```diff
@@ -0,0 +1,14 @@
+.product--rendered-attribute .form-item {
```

Comment:
This file should be commerce_product.rendered-attributes.css and have a docblock on top.